services/QuillLMS/client/app/bundles/Diagnostic/modules/analytics/index.ts

```diff
@@ -61,17 +76,6 @@ class SegmentAnalytics {
     }
   }
 
-  formatCustomProperties(properties: object): object {
-    if (typeof properties != 'object') {
-      properties = {};
-    }
-    return Object.keys(properties).reduce((accumulator, key) => {
-      let customKeyName = `custom_${key}`;
-      accumulator[customKeyName] = properties[key];
-      return accumulator;
-    }, {});
-  }
```

Comment:
As long as this is working in testing on staging, I don't think there'll be a problem, but now that I look at this code again I vaguely remember that we had to prepend `custom_` to avoid colliding with some platform's reserved keys list.  (Like, some platform wouldn't let you provide `user` or something.  But, like I said, if this is working in testing, I wouldn't worry about it.)

Reply:
I'll add this back in and just add a check to skip the `user_id` and `properties` keys